pkgs/development/python-modules/pytablewriter/default.nix

```diff
@@ -0,0 +1,60 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+
+, dataproperty
+, mbstrdecoder
+, pathvalidate
+, setuptools
+, tabledata
+, tcolorpy
+, typepy
+
+, pytestCheckHook
+, pyyaml
+, toml
+, elasticsearch
+, dominate
+}:
+
+buildPythonPackage rec {
+  pname = "pytablewriter";
+  version = "0.64.2";
+
+  src = fetchFromGitHub {
+    owner = "thombashi";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-+IOHnmdd9g3SoHyITJJtbJ0/SAAmwWmwX5XeqsO34EM=";
+  };
+
+  propagatedBuildInputs = [
+    dataproperty
+    mbstrdecoder
+    pathvalidate
+    setuptools
```

Comment:
setuptools shall not be propagated in most cases as it is a build-time only dependency. And it might not be required in the first place, as the default for `buildPythonPackage` is to treat the package as setuptools based, adding necessary dependencies.